### ISSUE TITLE
Python should default to python3 when not using conda??

### ISSUE THREAD

**Issue by umesh-timalsina** [deepforge-dev/deepforge#1673]
Currently, when not using conda, `start.js`  grabs system python. Should this be default to `python3` instead of python? Because in most linux distros, `python` still symlinks to `python2.7`.

```py
$ python
Python 2.7.17 (default, Nov  7 2019, 10:07:09) 
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 

```

**Comment by brollb** [comment]
Yeah, this is probably a good idea. I get spoiled with bleeding edge distros ;) Regardless, the `python3` alias won't cause any issues on these distros either so it is probably a good idea.
